### PR TITLE
print subprocess error message when capture_output=True

### PR DESCRIPTION
**Pull request by syuu1228**
We currently does not able to get any error message from subprocess when we specified capture_output=True on subprocess.run().
This is because CalledProcessError does not print stdout/stderr when it raised, and we don't catch the exception, we just let python to cause Traceback.
Result of that, we only able to know exit status and failed command but
not able to get stdout/stderr.

To resolve this, add wrapper function "out()" for capture output, and
print stdout/stderr with error message inside the function.

This is scylla-machine-image version of https://github.com/scylladb/scylla/pull/10391

### CODE REVIEW

File: lib/scylla_cloud.py [hunk 1]
```diff
@@ -21,6 +21,21 @@
 from abc import ABCMeta, abstractmethod
 from lib.scylla_cloud_io_setup import aws_io_setup, gcp_io_setup, azure_io_setup
 
+
+def out(cmd, shell=True, timeout=None, encoding='utf-8', ignore_error=False):
+    res = run(cmd, capture_output=True, shell=shell, timeout=timeout, check=False, encoding=encoding)
+    if not ignore_error and res.returncode != 0:
+        print(f'Command \'{cmd}\' returned non-zero exit status: {res.returncode}')
```
Comment (benipeled) [inline lib/scylla_cloud.py:28]: Not critical but maybe here we can catch the `CalledProcessError` exception and use the `CalledProcessError.output`

File: lib/scylla_cloud.py [hunk 2]
```diff
@@ -21,6 +21,21 @@
...
+        print('----------  stdout  ----------')
+        print(res.stdout, end='')
+        print('------------------------------')
+        print('----------  stderr  ----------')
+        print(res.stderr, end='')
+        print('------------------------------')
```
Comment (benipeled) [inline lib/scylla_cloud.py:34]: You can simplfy it with
```python
print('''f
----------  stdout  ----------
{res.stdout}
.
.
.

''')

File: lib/scylla_cloud.py (hunk 1 above)
Comment (syuu1228) [reply lib/scylla_cloud.py]: switched to catching CalledProcessError exception.

File: lib/scylla_cloud.py (hunk 2 above)
Comment (syuu1228) [reply lib/scylla_cloud.py]: Switched to here-document style.

**Comment by syuu1228** [comment]
Rebased with latest next, and fixed code style things pointed out by @benipeled.

**Comment by syuu1228** [comment]
Rebased with next

**Comment by syuu1228** [comment]
@yaronkaikov ping

**Pull request merged**